internal/backends/postgresql/collection.go

```diff
@@ -261,13 +261,15 @@ func (c *collection) DeleteAll(ctx context.Context, params *backends.DeleteAllPa
 func (c *collection) Explain(ctx context.Context, params *backends.ExplainParams) (*backends.ExplainResult, error) {
 	p, err := c.r.DatabaseGetExisting(ctx, c.dbName)
 	if err != nil {
-		return nil, lazyerrors.Error(err)
+		return &backends.ExplainResult{
```

Comment:
Isn't it simpler to handle `nil` in handler code? :thinking:

Reply:
That was my first approach, but after some time I realized we don't return nil in sqlite, so I've taken suggestion from there.

Reply:
All return non nil response.